odoo/fields.py

```diff
@@ -705,6 +705,8 @@ def resolve_deps(self, model, path0=[], seen=frozenset()):
                 _logger.warning("Field %s depends on itself; please fix its decorator @api.depends().", self)
             model, path = model0, path0
             for fname in dotnames.split('.'):
+                if fname not in model._fields:
+                    raise KeyError("Field : %s, wrong field '%s' of this deps : '%s'." % (self, fname, dotnames)
```

Comment:
missing brace + spelling

```suggestion
                    raise KeyError("Field %s depends on unknown field %s" % (self, fname))
```